xc/common/cmake/arch_define.cmake

```diff
@@ -75,7 +75,10 @@ function(ADD_XC_ARCH_DEFINE)
       --incremental_reroute_delay_ripup off \
       --base_cost_type delay_normalized_length_bounded \
       --bb_factor 10 \
-      --initial_pres_fac 4.0 \
+      --acc_fac 0.7 \
+      --astar_fac 1.8 \
```

Comment:
An A* factor of 1.8 likely has a very bad effect on QoR.  What do your results show on CPD with these changes?

Reply:
The difference is very small:
baselitex: [48.1 MHz](https://hydra.vtr.tools/build/40238/download/1/meta.json) -> [47.9 MHz](https://hydra.vtr.tools/build/41245/download/1/meta.json) sys_clk, clk200_clk slowed to 202 MHz, but still meets timing. 150 seconds faster.
ibex: [32.8 MHz](https://hydra.vtr.tools/build/40172/download/1/meta.json) -> [30.7 MHz](https://hydra.vtr.tools/build/41220/download/1/meta.json) clk. 50 seconds faster.

Reply:
Well that means the lookahead is doing a good job.  Just keep in mind that increasing A* could result in dramatic QoR degradation if the lookahead mispredicts.

Reply:
Good to tune on more designs in the longer term, as 3 designs is a small number for fine tuning.

Reply:
Default initial_pres_fac is 0.5; pres_fac_mult is 1.3.

Reply:
So you pay more attention to congestion early, which makes sense that it helps runtime. You ramp the cost of congestion more slowly, which would probably partially compensate for a higher initial cost of congestion (may mitigate  quality losses). So the congestion schedule is just different, in a way that seems reasonable.